cdhweb/events/models.py

```diff
@@ -88,22 +90,25 @@ class Event(Displayable, RichText, AdminThumbMixin, ExcerptMixin):
                                  on_delete=models.SET_NULL)
     event_type = models.ForeignKey(EventType, on_delete=models.CASCADE)
     speakers = models.ManyToManyField(Person,
-        help_text='Guest lecturer(s) or Workshop leader(s)',
-        blank=True)
+                                      help_text='Guest lecturer(s) or Workshop leader(s)',
+                                      blank=True)
 
     attendance = models.PositiveIntegerField(null=True, blank=True,
-        help_text='Total number of people who attended the event. (Internal only, for reporting purposes.)')
+                                             help_text='Total number of people who attended the event. (Internal only, for reporting purposes.)')
+
+    url = models.URLField(
+        null=True, blank=True, help_text='URL for virtual events, e.g. Zoom meetings.')
```

Comment:
Yeah, maybe we should rename it `virtual_url` or `join_url` or something? The validation probably wouldn't hurt too, I'll try it

Reply:
I'm going to leave off validation in favor of updated help text for now, since in #200 we might add support for dual virtual/in-person events or other features that would interact with it.